src/components/LandingFilter.jsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-env jquery */
+import React, { useState, useEffect } from 'react';
+import { FiArrowRightCircle } from 'react-icons/fi';
+import {
+  Dropdown, Grid, Form
+} from 'semantic-ui-react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import theme from '../config/theme';
+
+const GoButton = styled(Link)`
+  font-size: 50px;
+  font-weight: bold;
+  color: ${theme.colours.maroon};
+  :hover {
+    color: ${theme.colours.maroon};
+    opacity: 0.8;
+  }
+`;
+
+const regionOptions = [
+  { key: 'North', text: 'North', value: 'North' },
+  { key: 'South', text: 'South', value: 'South' },
+  { key: 'East', text: 'East', value: 'East' },
+  { key: 'West', text: 'West', value: 'West' }
+];
+
+const paxOptions = [
+  { key: '1-5', text: '1-5', value: '1-5' },
+  { key: '6-10', text: '6-10', value: '6-10' },
+  { key: '> 11', text: '> 11', value: '> 11' }
+];
+
+const ratingOptions = [
+  { key: '5', text: '5', value: '5' },
+  { key: '4', text: '4', value: '4' },
+  { key: '3', text: '3', value: '3' },
+  { key: '2', text: '2', value: '2' },
+  { key: '1', text: '1', value: '1' }
+];
+
+const LandingFilter = ({ category } = {}) => {
+  const [values, setValues] = useState({
+    keyword: '',
+    region: '',
+    pax: '',
+    rating: '',
+    price: '100'
+  });
+  const onChange = (e, result) => {
+    const { name, value } = result || e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  useEffect(() => {
+    $(document).ready(() => {
```

Comment:
i don think we neeed the $(document).ready here cos useEffect runs both after the first render and after every update